tests/lib/rules/no-undef-init.js

```diff
@@ -27,17 +27,17 @@ ruleTester.run("no-undef-init", rule, {
     invalid: [
         {
             code: "var a = undefined;",
-            output: "var a;",
+            output: null,
             errors: [{ message: "It's not necessary to initialize 'a' to undefined.", type: "VariableDeclarator" }]
         },
         {
             code: "var a = undefined, b = 1;",
-            output: "var a, b = 1;",
+            output: null,
             errors: [{ message: "It's not necessary to initialize 'a' to undefined.", type: "VariableDeclarator" }]
         },
         {
             code: "var a = 1, b = undefined, c = 5;",
-            output: "var a = 1, b, c = 5;",
+            output: null,
```

Comment:
Can you add similar tests to the old versions, but with `let`? Previously, none of the tests used `let` because the rule worked the same way with `var` as it did with `let`. Now that there is a behavior change between the two types, we should make sure the normal autofixing behavior still works correctly with `let`. (For example, we should test that `let a;` gets fixed to `let a, b = 1` gets autofixed to `let a = undefined, b = 1`, and we should test that `let [a] = undefined` still doesn't get autofixed.)